CogNative/models/RTVC/RTVC.py

```diff
@@ -50,8 +54,33 @@ def encode_voice(self, file_path):
         # Use Encoder to create embedding of input audio
         embed_wav = audio.preprocess_wav(og_wav, sampling_rate)
         self.embed = encoder.embed_utterance(embed_wav)
-    
+        if save_embedding:
+            self.save_embedding()
+
+    def set_file_path(self, file_path):
+        # SET FILE PATH
+        self.file_path = file_path
+        in_wav = Path(self.file_path)
+        assert os.path.exists(in_wav), "ERROR: File not found."
```

Comment:
The ```file_path``` can be changed here. This may be more useful in the future, but for now I am using it in ```main.py``` to set the ```file_path``` without needing to call ```RTVC.encode_voice()```.